include/swift/Sema/Constraint.h

```diff
@@ -258,6 +258,12 @@ enum class ConversionRestrictionKind {
   /// Implicit conversion from an Objective-C class type to its
   /// toll-free-bridged CF type.
   ObjCTollFreeBridgeToCF,
+  /// Implicit conversion from a value of Double to a value of CGFloat type via
+  /// an implicit CGFloat initializer call.
+  DoubleToCGFloat,
+  /// Implicit conversion from a value of CGFloat type to a value of Double type
+  /// via an implicit call to an initializer passing CGFloat value.
```

Comment:
Is this just "via an implicit Double initializer call" or is the wording difference significant for some other nuance?

Reply:
I just wanted to point up what the argument is going to be in this case as well, since there are a lot of double initializer overloads out there.

Reply:
✅